keps/sig-scheduling/5004-dra-extended-resource/README.md

```diff
@@ -817,8 +815,8 @@ ensure `ExtendedResourceName`s are handled by the scheduler as described in this
 
 #### Beta
 
-- Reevaluate where to create the special resource claim, in scheduler or some
-  other controller, based on feedback from Alpha and the nomination concept.
+- The basic scoring in NodeResourcesFit has to be implemented and that the queueing hints have to work efficiently.
+- Keep the Alpha behavior to create the special resource claim in scheduler.
 - Gather feedback from developers and surveys
 - 3 examples of vendors making use of the extensions proposed in this KEP
```

Comment:
you can strike this, it's not really relevant here, this does not require vendor driver support or changes

Reply:
removed it.